doc/source/programs/gdal_sozip.rst

```diff
@@ -0,0 +1,267 @@
+.. _gdal_sozip_command:
+
+================================================================================
+"gdal sozip" command
+================================================================================
+
+.. versionadded:: 3.11
+
+.. only:: html
+
+    SOZIP (Seek-Optimized ZIP) related commands
+
+.. Index:: gdal sozip
+
+Description
+-----------
+
+The :program:`gdal sozip` utility can be used to:
+
+- create a :ref:`sozip_intro` file
+- append files to an existing ZIP/SOZip file
+- list the contents of a ZIP/SOZip file
+- validate a SOZip file
+- convert an existing Zip file in a SOZip optimized one
+
+Synopsis
+--------
+
+.. program-output:: gdal sozip --help-doc
+
+
+"gdal sozip create"
+-------------------
+
+Description
++++++++++++
+
+Adds one or several files to a new or existing zip file.
+
+Synopsis
+++++++++
+
+.. program-output:: gdal sozip create --help-doc
```

Comment:
"create" is not a perfect name if the target zip may exist already

Reply:
indeed. Most of the time it will be used for creation though. Any other suggestion? I thought to "add" but it doesn't cover the create use case. Or having both create and add doing the same... ? but that seems a bit overkill

Reply:
I don't know really. Pack, put, insert, write, or zip? Are they any better, I don't know.